sdk/nodejs/package.json

```diff
@@ -25,6 +25,7 @@
         "js-yaml": "^3.14.0",
         "minimist": "^1.2.6",
         "normalize-package-data": "^2.4.0",
+        "pkg-dir": "^7.0.0",
```

Comment:
If you think you can implement this more simply than the package does, go for it. (Eliding a security and supply chain argument for a second), IMO we shouldn't worry about # deps unless we have a compelling reason -- package size too big, load times too long, etc. For example, the code in this package uses another dependency, `findUp`, and implements its core logic in 2 lines. I suspect `findUp` does the directory traversal. I don't imagine we're going to find a lot of cruft there either. I'd be surprised if this dependency is at all significant to the metrics we care about.

Reply:
looks like `getPackageDefinition` indeed does something _similar_  but I'm a bit reluctant to reuse the function as-is in my PR, mainly due to:
1. it's currently a private function in the closure package and non-exported. I'd have to move it somewhere else.
2. It doesn't seem to have any tests, which brings me back to why I was using `pkg-dir` in the first place :). I know I could implement searching parent paths myself but testing this (this would require mocking IO or creating temp files) would involve some non-trivial amount of work. As @AaronFriel  called out `pkg-dir` and all it's transitive dependencies are written by @sindresorhus who is known to write high quality + well *tested* minimal packages.

Reply:
SGTM